src/renderers/shared/fiber/ReactFiberBeginWork.js

```diff
@@ -63,6 +63,8 @@ module.exports = function<T, P, I, TI, C>(
   scheduleUpdate : (fiber: Fiber) => void
 ) {
 
+  const { shouldSetTextContent } = config;
```

Comment:
This is what the second paragraph refers to. :) I meant the top level scope or the "config" scope.

Ideally config should just be `import { shouldSetTextContent } from "HostConfig";` and then "HostConfig" is injectable by the build tooling.